README.md

```diff
@@ -17,8 +17,9 @@
 ```
 Root project folder
 ├─ .gitignore
-├─ .env                                         - env variables, it mustn't be versioned/published
-├─ .env.dist                                    - template of the .env file
+├─ .env                                         - contains API host, should be versioned
```

Comment:
Takto by to malo by spravne. 
- `.env` public verziovane hodnoty.
- `.env.local` - secrets
- `.env.dist` sablona